pr_agent/git_providers/bitbucket_provider.py

```diff
@@ -173,14 +203,16 @@
     @staticmethod
     def _parse_pr_url(pr_url: str) -> Tuple[str, int]:
         parsed_url = urlparse(pr_url)
-        
-        if 'bitbucket.org' not in parsed_url.netloc:
+
+        if "bitbucket.org" not in parsed_url.netloc:
             raise ValueError("The provided URL is not a valid Bitbucket URL")
```

Comment:
**Suggestion:** Add error handling for the case where the Bitbucket PR URL is not valid.
```suggestion
        if "bitbucket.org" not in parsed_url.netloc:
            logging.error("The provided URL is not a valid Bitbucket URL")
            return None, None
```